jobs/build/ocp/Jenkinsfile

```diff
@@ -162,6 +162,58 @@ def get_mirror_url(build_mode, version) {
     return "https://mirror.openshift.com/enterprise/enterprise-${version}"
 }
 
+def get_changelog(rpm_name, record_log) {
+    //
+    // INPUTS:
+    //   rpm_name - the name of an RPM build previously
+    //   record_log - an array of build records with | separated fields
+
+    rpm_builds = record_log['build_rpm']
+    if (rpm_builds == null || rpm_builds.size() == 0) {
+        return ""
+    }
+
+    // filter for the desired RPM using name
+    build_record_index = rpm_builds.findIndexOf {
+        it['rpm'] == rpm_name
+    }
+    if (build_record_index == -1) {
+        return ""
+    }
+    build_record = rpm_builds[build_record_index]
+
+    // then get the task_id and task_url out of it
+    // task_id = build_record['task_id']
+    task_url = build_record['task_url']
+
+    // get the build ID from the web page
+    // there must be an API way to do this MAL 20180622
+    build_id = sh(
+        returnStdout: true,
+        script: [
+            "curl --silent --insecure ${task_url}",
+            "sed -n -e 's/.*buildID=\\([0-9]*\\).*/\\1/p'"
+        ].join(" | ")
+    ).trim()
+
+    // buildinfo can return the changelog.  Return just the text after
+    // the Changelog: line
+    try {
+        changelog = sh(
+            returnStdout: true,
+            script: [
+                "brew buildinfo ${build_id} --changelog",
+                "sed -n '/Changelog/,\$p'"
+            ].join(' | ')
+        ).trim()
+    } catch (err) {
```

Comment:
Above should be like this in terms of silently failing.

Reply:
Will address.